app/helpers/sortable_table_helper.rb

```diff
@@ -1,26 +1,37 @@
 module SortableTableHelper
-  def table_header(column, i18n_stem, index, path, sort_by, sort_direction)
-    if sort_by == column
-      aria_sort = sort_direction
-      next_direction = sort_direction == 'ascending' ? 'descending' : 'ascending'
-    else
-      aria_sort = 'none'
-      next_direction = 'descending'
-    end
+  def table_header(column, i18n_stem, index, path, sort_by, sort_direction, additional_classes: [])
+    aria_sort, next_direction = sort_variable(column, sort_by, sort_direction)
 
     next_direction = sort_direction == 'ascending' ? 'descending' : 'ascending'
-    tag.th(scope: 'col', class: 'govuk-table__header', 'aria-sort': aria_sort) do
+    tag.th(scope: 'col', class: ['govuk-table__header'] + additional_classes, 'aria-sort': aria_sort) do
       reorder_form(path, column, next_direction, i18n_stem, index)
     end
   end
 
   def reorder_form(path, column, next_direction, i18n_stem, index)
-    tag.form(action: path, method: 'get') do
-      safe_join([tag.input(type: 'hidden', name: 'sort_by', value: column),
-                 tag.input(type: 'hidden', name: 'sort_direction', value: next_direction),
-                 tag.button(type: 'submit', 'data-index': index) do
-                   I18n.t(column, scope: i18n_stem)
-                 end])
+    tag.form(action: path, params: params, method: 'get') do
```

Comment:
do we need/want the params in the URL here given this is a post request and we have a heap of form fields as well?

hard to know when this is used/what can appear here - at the very least I feel like we should explicitly pass in any params we want to use

Reply:
This is specifically to handle the reused 'allowed_` logic of the view claims controller via the prefix param, at line 14. will have a look at passing the specific param but it feels more flexible/extensible to pass all and use the ones we want to me

Reply:
ahh, have removed the params from the form tag as it is not needed there. instead we can just access params['prefix'] given the view context for the helper